### ISSUE TITLE
Neutron bomb

### ISSUE THREAD

**Issue by Drogo-man** [studio-minus/ppg-bugs-and-requests#2359]
**Feature description**
A very huge bomb that vaporizes everything on the map.

**Feature purpose**
Sometimes "clear everything" can be a bit boring.
Also big boom is good.

**Comment by DindinYT37** [comment]
> Sometimes "clear everything" can be a bit boring.

Not really a valid purpose.

**Comment by Drogo-man** [comment]
Yes, for people who tolerate fun.

**Comment by DindinYT37** [comment]
> Yes, for people who tolerate fun.

Indeed, it's not a valid purpose for people who tolerate fun, either.

**Comment by Drogo-man** [comment]
As if you would know.

**Comment by DindinYT37** [comment]
> As if you would know.

You just told me, so if you were trying to imply my statement was incorrect, you've actually just said that you lied and wouldn't know.

**Comment by Drogo-man** [comment]
I have no clue what the hell you mean.

**Comment by username90001** [comment]
neutron bomb is more like thing that will create a cyan flash and after make everyone get tritium poison

**Comment by username90001** [comment]
irl neutron bombs are small nuclear devices that used instead of destroying they used for create neutron radioation that kills everything near it + cancer

**Comment by username90001** [comment]
ways to do something like neutron bomb 1 place alot of shower things connect them to tritium source and power up shower thing that i forgot or way 2 nuclear playground
